packages/client/CHANGELOG.md

```diff
@@ -6,6 +6,37 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.3.0 - 2022-01-20
+
+### New RPC Endpoints: eth_getLogs, eth_getTransactionReceipt and eth_getTransactionByHash
+
+This release adds receipt and log saving functionality to the client. There has been a new database infrastructure laid out in PR [#1556](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1556) for data, caching and indexes.
+
+A new `ReceiptsManager` handles receipt and log saving and lookup, along with the `txHash -> [blockHash, txIndex]` index that is limited by `--txLookupLimit` (default = 2350000 blocks = about one year, 0 = entire chain).
+
+Enable with `--saveReceipts` and use with --rpc to enable `eth_getLogs`, `eth_getTransactionReceipt` and `eth_getTransactionByHash` endpoints.
+
+### Sepolia PoW Testnet Support
+
+The client now support the new [Sepolia](https://sepolia.ethdevops.io/) testnet, which is a PoW network intended to replace the coming-to-ages `ropsten` network, see PR [#1581](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1581).
```

Comment:
```suggestion
The client now supports the new [Sepolia](https://sepolia.ethdevops.io/) testnet, which is a PoW network intended to replace the `ropsten` network, see PR [#1581](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1581).
```

Reply:
didn't really understand what you meant by "coming-to-ages" 😄

Reply:
That's a direct translation of the german "in die Jahre kommen" which basically means "getting old". 😋 Might not exist in english. 😅